private-templates-service/client/src/components/Common/SpinnerModal/styled.tsx

```diff
@@ -0,0 +1,16 @@
+import styled from 'styled-components';
+import { Spinner } from 'office-ui-fabric-react/lib/Spinner';
+import { COLORS } from '../../../globalStyles';
+
+export const CenteredSpinner = styled(Spinner)`
+  margin: auto;
+`;
+
+export const Dot = styled.div`
```

Comment:
What does this do? I saw that it wraps the spinner.

Reply:
Is is just a white circular background that the spinner can sit on top off?

Reply:
yes